### PR TITLE
Added solution HW-1

### PR DESCRIPTION
**Pull request by aborozdykh**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        boolean fullMatch = result1 == bet1 && result2 == bet2;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: try to solve it without redundant variables https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Comment (aborozdykh) [reply src/main/java/core/basesyntax/FootballMatch.java]: Done.

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: '?' should be on a new line

Comment (aborozdykh) [reply src/main/java/core/basesyntax/FootballMatch.java]: Done.

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**